### PR TITLE
Release 2.0 documentation

### PR DESCRIPTION
**Pull request by eht16**
Add NEWS for 2.0.

Remarks for the NEWS items:
- I skipped many smaller or non user visible changes or follow-up PRs to not bloat the list too much
- if anyone miss his favorite PR or so, shout out

### CODE REVIEW

**Comment by elextr** [comment]
LGTM.

File: NEWS [hunk 1]
```diff
@@ -1,6 +1,57 @@
...
+    * Update documentation for "geany.keygrab" and fix infinite loop
+      (xiota, PR #1236)
+    * Update for Scintilla 5.1.4 (xiota, PR #1123)
+    * Update glspi_keycmd.h: add GEANY_KEYS_FILE_RELOAD_ALL, remove
+      GEANY_KEYS_COUNT (Skif-off, PR #1124)
+    * Add geany.settype() to set the document filetype (Skif-off, PR #1157)
+
+    GeanyVC:
+    * Support the use of external diff-tool in directory context
+     (nomadbyte, PR #1025)
```
Comment (frlan) [inline NEWS:23]: Wrong indent

Comment (eht16) [reply NEWS]: Fixed, thanks.

**Comment by frlan** [comment]
LGTM

**Pull request merged**